docs/src/app/components/markdown/lists-markdown.md

```diff
@@ -0,0 +1,31 @@
+# Headline 1
+
+This is a paragraph describing something.
+
+## Subheadline
+
+Detailes description here. And some code:
+```javascript
```

Comment:
From experience, npm works best with just `js`. Should we follow this conventio?